main.tex

```diff
@@ -695,10 +699,11 @@ \subsubsection*{Provenance entities}
 \textbf{\RQ{} [\texttt{RQ}]:}
 
 \noindent The research question (or research objective or problem formulation) determines the goal of the research presented in a publication.
-In the case of simulation studies, it typically forms the starting point in the modeling and simulation life cycle~\cite{Balci2012, Robinson2008} and is central to interpret its products such as simulation data or simulation model.
+For simulation studies, it typically forms the starting point of the modeling and simulation life cycle~\cite{Balci2012, Robinson2008} and is key to interpreting its outputs such as simulation data or a simulation model.
 
-As for our provenance example shown in Fig~\ref{fig:ProvLee}, \lee{}~\cite{Lee2003} questioned in \texttt{RQ1} the necessity of \enquote{the two scaffold proteins, APC and Axin} (for \wnt{} signaling) and whether \enquote{their roles differ}.
+As for the provenance example shown in Fig~\ref{fig:ProvLee}, \lee{}~\cite{Lee2003} questioned in \texttt{RQ1} the necessity of \enquote{the two scaffold proteins, APC and Axin} (for \wnt{} signaling) and whether \enquote{their roles differ}.
 This research question determines a minimum number of model constituents and guides the modeler towards possible simulation experiments to be executed.
```

Comment:
Not entirely sure what is meant by "model constituents"? I think this might just be my fault though so I didn't actually make any changes